test-app/docker-compose.yml

```diff
@@ -1,9 +1,11 @@
-name: marklogic_python
+# name: docker-tests-marklogic_python
+name: docker-tests-marklogic_python-12
```

Comment:
Remove the "-12" from here - let's just make 12 the standard now, you can remove the commented out image name below as well.